network/newstream/common_test.go

```diff
@@ -63,3 +205,111 @@ func newTestLocalStore(id enode.ID, addr *network.BzzAddr, globalStore mock.Glob
 	}
 	return localStore, cleanup, nil
 }
+
+func parseID(str string) ID {
+	v := strings.Split(str, "|")
+	if len(v) != 2 {
+		panic("too short")
+	}
+	return NewID(v[0], v[1])
+}
+
+func uploadChunks(ctx context.Context, store chunk.Store, count uint64) (chunks []chunk.Address, err error) {
+	for i := uint64(0); i < count; i++ {
+		c := storage.GenerateRandomChunk(4096)
+		exists, err := store.Put(ctx, chunk.ModePutUpload, c)
+		if err != nil {
+			return nil, err
+		}
+		if exists[0] {
+			return nil, errors.New("generated already existing chunk")
+		}
+		chunks = append(chunks, c.Address())
+	}
+	return chunks, nil
+}
+
+func mustUploadChunks(ctx context.Context, t testing.TB, store chunk.Store, count uint64) (chunks []chunk.Address) {
```

Comment:
I think in `must` functions, generally `t` should be the first argument.

Reply:
That would be expected, but linter complains about context not being the first argument.